### PR TITLE
DIV-6815: Hide two tabs from solicitor users

### PR DESCRIPTION
**Pull request by jvictorcf**
### Change description ###
I've made the change in production because otherwise solicitors would have access to these tabs now and when share-a-case goes live. Jeremy has agreed with the approach.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: definitions/divorce/json/CaseTypeTab/CaseTypeTab.json [hunk 1]
```diff
@@ -3464,50 +3464,204 @@
     "TabFieldDisplayOrder": 7
   },
   {
-    "LiveFrom": "12/12/2019",
+    "LiveFrom": "01/03/2021",
```
Comment (johnoneill46) [inline definitions/divorce/json/CaseTypeTab/CaseTypeTab.json:3467]: I don't think liveFrom works anymore does it?

**Review by yarpo — APPROVED** [review]

File: definitions/divorce/json/CaseTypeTab/CaseTypeTab.json (hunk 1 above)
Comment (jvictorcf) [reply definitions/divorce/json/CaseTypeTab/CaseTypeTab.json:3467]: I'm not sure.

File: definitions/divorce/json/CaseTypeTab/CaseTypeTab.json [hunk 2]
```diff
@@ -3464,50 +3464,204 @@
...
     "CaseTypeID": "DIVORCE",
     "Channel": "CaseWorker",
-    "TabID": "serviceApplication",
+    "TabID": "serviceApplicationCourtAdmin",
```
Comment (Dizrupto) [inline definitions/divorce/json/CaseTypeTab/CaseTypeTab.json:3470]: Should all those changes be in a feature toggled file?

File: test/unit/definitions/divorce/CaseTypeTab.test.js [hunk 3]
```diff
@@ -205,12 +215,19 @@ describe('CaseTypeTab (prod)', () => {
     expect(validationErrors).to.have.lengthOf(0);
   });
 
-  it('should be visible to specific users', () => {
-    assertTabIsVisibleForGivenUserRole(caseTypeTab, 'History', caseworkerBetaUserRole);
-    assertTabIsVisibleForGivenUserRole(caseTypeTab, 'Payment', caseworkerBetaUserRole);
+  it('should be visible to user roles', () => {
+    assertTabIsVisibleForGivenUserRoles(caseTypeTab, 'Service Application', userRoles.allHumanNonSolicitorCcdUiUserRoles);
+    assertTabIsVisibleForGivenUserRoles(caseTypeTab, 'General Referral', userRoles.allHumanNonSolicitorCcdUiUserRoles);
```
Comment (Dizrupto) [inline test/unit/definitions/divorce/CaseTypeTab.test.js:220]: Nice :)

File: definitions/divorce/json/CaseTypeTab/CaseTypeTab.json (hunk 1 above)
Comment (johnoneill46) [reply definitions/divorce/json/CaseTypeTab/CaseTypeTab.json:3467]: https://tools.hmcts.net/confluence/display/RCCD/CCD+Definition+Glossary+for+Setting+up+a+Service+in+CCD#CCDDefinitionGlossaryforSettingupaServiceinCCD-ccdDefTabCaseTypeTab

sorry my mistake - 'liveTo' doesn't work anymore - live from seemingly seems to still work

**Review by Dizrupto — APPROVED** [review]

**Pull request merged**